docs/src/content/docs/fr/reference/overrides.md

```diff
@@ -0,0 +1,374 @@
+---
+title: Référence des redéfinitions
+description: Une vue d'ensemble de tous les composants et les props des composants supportés par les redéfinitions de Starlight.
+tableOfContents:
+  maxHeadingLevel: 4
+---
+
+Vous pouvez redéfinir les composants intégrés à Starlight en spécifiant des chemins vers des composants de remplacement avec l'option de configuration [`components`](/fr/reference/configuration#components) de Starlight.
+Cette page répertorie tous les composants disponibles qui peuvent être redéfinis et fournit des liens vers leurs implémentations par défaut sur GitHub.
+
+Pour en savoir plus, consultez le [guide des redéfinitions de composants](/fr/guides/overriding-components/).
+
+## Props des composants
```

Comment:
Using "props" like the [Astro docs](https://docs.astro.build/fr/core-concepts/astro-components/#props-de-composant) for consistency.